raphtory/src/core/entities/properties/props.rs

```diff
@@ -275,6 +275,28 @@ impl PropMapper {
             dtypes: Arc::new(RwLock::new(dtypes)),
         }
     }
+
+    pub fn get_and_validate(
+        &self,
+        prop: &str,
+        dtype: PropType,
+    ) -> Result<Option<usize>, GraphError> {
+        match self.get_id(prop) {
+            Some(id) => {
+                let existing_dtype = self.get_dtype(id).unwrap();
```

Comment:
I know this cannot fail, but worth turning the unwrap into a '?'

Reply:
Can turn it into an expect, if this is ever not true, we want shit to blow up!